sdk/servicebus/azure-servicebus/azure/servicebus/management/_constants.py

```diff
@@ -5,14 +5,32 @@
 
 API_VERSION_PARAM_NAME = "api-version"
 API_VERSION = "2017-04"
-ENTRY_TAG = "{http://www.w3.org/2005/Atom}entry"
-CONTENT_TAG = "{http://www.w3.org/2005/Atom}content"
-QUEUE_DESCRIPTION_TAG = "{http://schemas.microsoft.com/netservices/2010/10/servicebus/connect}QueueDescription"
-COUNT_DETAILS_TAG = "{http://schemas.microsoft.com/netservices/2010/10/servicebus/connect}CountDetails"
-TITLE_TAG = "{http://www.w3.org/2005/Atom}title"
-
 ENTITY_TYPE_QUEUES = "queues"
 ENTITY_TYPE_TOPICS = "topics"
-
 LIST_OP_SKIP = "$skip"
 LIST_OP_TOP = "$top"
+
+XML_SCHEMA_NAMESPACE = "http://www.w3.org/2001/XMLSchema"
+XML_SCHEMA_INSTANCE_NAMESPACE = "http://www.w3.org/2001/XMLSchema-instance"
+
+SB_XML_NAMESPACE = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+ENTRY_TAG = "{http://www.w3.org/2005/Atom}entry"
+CONTENT_TAG = "{http://www.w3.org/2005/Atom}content"
+
+RULE_KEY_VALUE = "KeyValueOfstringanyType"
+RULE_KEY = "Key"
+RULE_VALUE = "Value"
+RULE_VALUE_TYPE = "type"
+RULE_VALUE_TYPE_XML_PREFIX = "d6p1"
+SQL_COMPATIBILITY_LEVEL = "20"
```

Comment:
May be a TINY bit more explicit that this has to do with RULE_SQL_COMPATIBILITY_LEVEL.

Similarly, I may make tiny little comment headers above the blocks of constants to say "these constants all have to do with X or Y" for instance.  (Like, I see 'content_tag' above, to give one random instance, is this generic?  have to do with rules? xml schemas?  I could grep for it but even something like # Service-Bus wide XML tags (and grouping accordingly) would help understand-ability)

Reply:
Good suggestion.
I changed them to RULE_SQL_COMPATIBILITY_LEVEL, ATOM_CONTENT_TAG and ATOM_ENTRY_TAG. The two ATOM tags are generic.
Let me know if you have other constant name suggestions.

Reply:
The rest of the names look cogent++  I still am partial to having comment snippets above each block of constants to explain what section of logic they refer to but it's reasonably clear as is so take that or leave it.  Thanks for the tweaks :)

Reply:
I added some simple comments to each block of constants.